### PR TITLE
remove flux packaging.

### PR DESCRIPTION
**Pull request by ohadlevy**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.15 (optional)
* [ ] 1.14
* [ ] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ohadlevy** [comment]
test fails as nightly currently fails due to #1684

**Comment by mmoll** [comment]
[test]

File: foreman/foreman.spec [hunk 1]
```diff
@@ -588,6 +584,7 @@ Requires: %{?scl_prefix}rubygem(jquery_pwstrength_bootstrap_4) >= 1.2
 Requires: %{?scl_prefix}rubygem(jquery_pwstrength_bootstrap_4) < 2.0
 Requires: %{?scl_prefix}rubygem(jquery-turbolinks) >= 2.1
 Requires: %{?scl_prefix}rubygem(jquery-turbolinks) < 3.0
+Obsoletes: npm(flux)
```
Comment (dLobatog) [inline foreman/foreman.spec]: I don't think this is location correct, it should be `Obsoletes: npm(flux) < 2.1.1-2` on `tfm/tfm.spec` - the foreman package does not obsolete this.

**Comment by ehelms** [comment]
@ohadlevy ping

**Comment by mmoll** [comment]
@ohadlevy any update here?

**Comment by ohadlevy** [comment]
git annex is not installable on my F26, attach is the diff with latest changes, appreciate if you can squash on merge.
```diff
diff --git a/comps/comps-foreman-rhel7.xml b/comps/comps-foreman-rhel7.xml
index 9b06766eb..a4996dc3c 100644
--- a/comps/comps-foreman-rhel7.xml
+++ b/comps/comps-foreman-rhel7.xml
@@ -187,7 +187,6 @@
       <packagereq type="default">nodejs-extract-text-webpack-plugin</packagereq>
       <packagereq type="default">nodejs-fbjs</packagereq>
       <packagereq type="default">nodejs-file-loader</packagereq>
-      <packagereq type="default">nodejs-flux</packagereq>
       <packagereq type="default">nodejs-ipaddr.js</packagereq>
       <packagereq type="default">nodejs-jquery</packagereq>
       <packagereq type="default">nodejs-jquery-flot</packagereq>
diff --git a/foreman/foreman.spec b/foreman/foreman.spec
index 43be2c54f..3c97185ae 100644
--- a/foreman/foreman.spec
+++ b/foreman/foreman.spec
@@ -606,7 +606,6 @@ Requires: %{?scl_prefix}rubygem(jquery_pwstrength_bootstrap_4) >= 1.2
 Requires: %{?scl_prefix}rubygem(jquery_pwstrength_bootstrap_4) < 2.0
 Requires: %{?scl_prefix}rubygem(jquery-turbolinks) >= 2.1
 Requires: %{?scl_prefix}rubygem(jquery-turbolinks) < 3.0
-Obsoletes: npm(flux)
 
 %description assets
 Meta package to install asset pipeline support.
diff --git a/rel-eng/tito.props b/rel-eng/tito.props
index beeaea903..a1e1b923c 100644
--- a/rel-eng/tito.props
+++ b/rel-eng/tito.props
@@ -37,7 +37,6 @@ whitelist = foreman-bootloaders-redhat
   nodejs-extract-text-webpack-plugin
   nodejs-fbjs
   nodejs-file-loader
-  nodejs-flux
   nodejs-ipaddr.js
   nodejs-jquery
   nodejs-jquery.cookie
diff --git a/tfm/tfm.spec b/tfm/tfm.spec
index 1a1a78f90..d692c5823 100644
--- a/tfm/tfm.spec
+++ b/tfm/tfm.spec
@@ -56,6 +56,7 @@ Obsoletes: ruby193-ruby-wrapper
 
 # Obsolete packages no longer carried within this SCL to both clean up
 # and to ensure smooth upgrades when old packages deps aren't satisified
+Obsoletes: %{scl_prefix}npm(flux) < 2.1.1-2
 Obsoletes: %{scl_prefix}rubygem-ace-rails-ap < 4.1.1-2
 Obsoletes: %{scl_prefix}rubygem-ansi < 1.4.3-7
 Obsoletes: %{scl_prefix}rubygem-archive-tar-minitar < 0.5.2-12
```

**Comment by ohadlevy** [comment]
@ekohl thanks! - pushed the diff above and rebased.

**Comment by dLobatog** [comment]
@ohadlevy I think this still requires removing it from the Foreman spec

**Pull request merged**